### ISSUE TITLE
Use pre-existing gmails and not ones from gmailnator

### ISSUE THREAD

**Issue by derelictpillows** [NightfallGT/Discord-Account-Generator#23]
Is there a way to use a list of gmails already existing and using those for account creation, instead of generating temporary ones from gmailnator?

**Comment by AbdelrahmanMostafaSamy** [comment]
pretty sure there isnt an actual option for it but you can always modify the code

**Comment by ICUBICI** [comment]
yeah you just get emails from a text file BUT you wont be able to really verify them

**Comment by FeelsBadMan1** [comment]
> yeah you just get emails from a text file BUT you wont be able to really verify them

Well you can use their imap sever to verify them. however what is more easy is to use beautifulsoup and a working tempmail like https://inbox.wtfbruh.nl
